dask/dataframe/dask_expr/io/tests/test_distributed.py

```diff
@@ -65,4 +65,6 @@ def test_pickle_size(tmpdir, filesystem):
     df = read_parquet(tmpdir, filesystem=filesystem)
     from distributed.protocol import dumps
 
-    assert len(b"".join(dumps(df.optimize().dask))) <= 10_000
+    assert (low_level := len(b"".join(dumps(df.optimize().dask)))) <= 10_000
+    # FIXME: The expression should be smaller than the raw graph
+    assert len(b"".join(dumps(df.optimize()))) <= low_level
```

Comment:
Two reasons for this (I'll probably not fix both before merging)

1. Fragments are not wrapped
2. Dataset object is attached to the expression but not actually required for materialization

Reply:
ok, I dug into this and fixed it in https://github.com/dask/dask/pull/11797